### PR TITLE
static pages

### PR DESCRIPTION
**Pull request by nguyenngoctruong**

### CODE REVIEW

File: app/views/static_pages/home.html.erb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<% provide(:title, "Home") %>
+  <div class="center jumbotron">
```
Comment (dieunb) [inline app/views/static_pages/home.html.erb]: indent

File: app/views/static_pages/help.html.erb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<% provide(:title, "Help") %>
+<!DOCTYPE html>
```
Comment (dieunb) [inline app/views/static_pages/help.html.erb]: html???

File: app/views/static_pages/about.html.erb [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<% provide(:title, "About") %>
+<!DOCTYPE html>
```
Comment (dieunb) [inline app/views/static_pages/about.html.erb]: html???

File: app/helpers/static_pages_helper.rb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
```
Comment (dieunb) [inline app/helpers/static_pages_helper.rb]: remove

File: app/helpers/application_helper.rb [hunk 5]
```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+	def full_title(page_title = '')
```
Comment (dieunb) [inline app/helpers/application_helper.rb]: indent

Comment (nguyenduyta) [inline app/helpers/application_helper.rb]: Indent

File: app/helpers/static_pages_helper.rb [hunk 6]
```diff
@@ -0,0 +1,2 @@
...
+end
```
Comment (nguyenduyta) [inline app/helpers/static_pages_helper.rb]: Deleted

File: app/views/layouts/_footer.html.erb [hunk 7]
```diff
@@ -0,0 +1,13 @@
+<footer class="footer">
+  <small>
+    The <a href="">Framgia E-Learning System</a>
+    by <a href="">Tung and Truong</a>
+  </small>
+  <nav>
+    <ul>
+      <li><%= link_to "About",   about_path %></li>
```
Comment (nguyenduyta) [inline app/views/layouts/_footer.html.erb]: Indent

File: app/views/layouts/_header.html.erb [hunk 8]
```diff
@@ -0,0 +1,12 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to "E-Learning", root_path, id: "logo" %>
+    <nav>
+      <ul class="nav navbar-nav navbar-right">
+        <li><%= link_to "Home",   root_path %></li>
+        <li><%= link_to "Help",   help_path %></li>
```
Comment (nguyenduyta) [inline app/views/layouts/_header.html.erb]: Indent

File: app/views/static_pages/about.html.erb [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+<html>
+
+  <body>
```
Comment (nguyenduyta) [inline app/views/static_pages/about.html.erb]: ??? deleted
<!DOCTYPE html>
+<html>
+
+  <body>

**Pull request merged**